apex/pyprof/README.md

```diff
@@ -117,22 +128,28 @@ For both Tensor Core and non-Tensor Core Deep Learning performance optimization
 
 ### TODOs
 1. The support for conv transpose is currently missing.
-2. PyProf currently works only with NvProf, but Nsight Compute support will be added in the future.
+2. PyProf currently works with NvProf and NSight Systems. Nsight Compute support will be added in the future.
```

Comment:
I am not sure if we need separate support for Nsight Systems and Nsight Compute. Please verify with the Nsight team.

Reply:
Good point re Nsight Compute. In any case, that's a TODO issue, rather than something critical to the PR being functional.

Reply:
I would leave it for now. I don't know which direction we plan to take pyprof support here.